index.html

```diff
@@ -95,8 +96,12 @@ <h3>Multi-Post Stories</h3>
           <li>boostrap</li>
           <li>ruby</li>
         </ul>        
-        <!-- <div class="button_works"> -->
-          <form class="button_works " action="https://github.com/diegoh40">
+        
+          <!-- <form class="button_works " action="https://github.com/diegoh40">
+            <button type="submit">See Project</button>
+          </form> -->
```

Comment:
- [ ] please try to remove unused commented code in order to keep your file professional